connectors/byoei.py

```diff
@@ -104,8 +106,10 @@ def _bulk_op(self, doc, operation=OP_INDEX):
 
         raise TypeError(operation)
 
-    async def _batch_bulk(self, task_num, operations):
+    async def _batch_bulk(self, operations):
         # todo treat result to retry errors like in async_streaming_bulk
+        task_num = len(self.bulk_tasks)
```

Comment:
Is this the pythonic way to do this sort of thing? I would have expected `bulk_tasks.size()` or `bulk_tasks.count()` as opposed to this more functional approach.

Reply:
yes, that's the Pythonic way. In Python, container classes implement `__len__` and that gives you `len(object)`
But `.size()` or `.count()` is also something you would see.